libs/service-portal/graphql/src/lib/queries/getOrganizations.ts

```diff
@@ -0,0 +1,24 @@
+import { gql } from '@apollo/client'
+
+const OrganizationFragment = gql`
+  fragment OrganizationFragment on Organization {
+    id
+    title
+    logo {
+      id
+      url
+      title
+    }
+  }
+`
+
+export const GET_ORGANIZATIONS_QUERY = gql`
+  query getOrganizations {
+    getOrganizations(input: { lang: "is-IS", perPage: 200 }) {
```

Comment:
Should revisit, getting all orgs when we'll only display a fraction isn't ideal.
But, we're getting all documents and fake-paging.. so we know the whole list of document providers.
Cache in redis?